### DISCUSSION TITLE
how to get the pause frame of the video through ui.video

### DISCUSSION THREAD

**Discussion by Zhaogaog** [zauberzeug/nicegui#1891] [Q&A] [answered]
### Question

I'm new to Nicegui, I'm displaying a local video via ui.video, and I'm now able to add a button to pause the video by clicking the "pause button". When the video is paused, I want to get the current frame of the video as an image input and then do some image processing. How can I get the paused frame of the video or the video frame at any given time or the video frame with a specific serial number? I sincerely ask you all and look forward to your replies, thank you very much!

**Comment by falkoschindler** [comment]
Crosspost: https://discord.com/channels/1089836369431498784/1167062819452231710/1167086904244715530

**Comment by rodja** [comment, marked as answer]
As has been written on Discord, you can get the current time

```py
current_time = await ui.run_javascript("document.querySelector('video').currentTime")
frame = math.floor(current_time * fps)
```

Then use ffmpeg or similar to extract the frame from the video.
